src/Symfony/Component/Uid/Uuid.php

```diff
@@ -60,17 +60,16 @@ public static function fromString(string $uuid): parent
             return new static($uuid);
         }
 
-        if (!uuid_is_valid($uuid)) {
-            throw new \InvalidArgumentException(sprintf('Invalid UUID%s: "%s".', static::TYPE ? 'v'.static::TYPE : '', $uuid));
+        if ('00000000-0000-0000-0000-000000000000' === $uuid) {
```

Comment:
The `Ulid` class is using a constant for this value. I assume you dont do that here because it is only used once. Should we care about constancy? 

Or maybe a public constant on `NilUuid`.

Reply:
We don't want to open the API for that, so no public thing. They yes, it's used once, there is nothing to save with a const here.

Reply:
I now added the const, since I found other places where the value could be used :)